.circleci/config.yml

```diff
@@ -59,7 +59,53 @@ jobs:
                   name: Build bot
                   command: cd frontend/bot && yarn run build-ci
 
-    
+    build-archive:
+        resource_class: xlarge
+        docker:
+            - image: codaprotocol/coda:toolchain-54430467ba429af285ea937d1c1da7d4b4cbde3e
+              environment:
+                CODA_DOCKER: true
+                HASURA_PORT: 8080
+            - image: postgres:12
+              environment:
+                POSTGRES_PASSWORD: codarules
+                POSTGRES_USER: admin
+                POSTGRES_DB: archiver
+            - image: hasura/graphql-engine:v1.0.0-beta.6
+              entrypoint: ["sh", "-c"]
+              command: ["sleep 20 && graphql-engine serve"]
```

Comment:
Is it possible to drop this to 5 seconds?

Reply:
It doesn't affect the ultimate build-time, I found 10 seconds was sufficient but doubled that out of an abundance of caution.

Reply:
ah let's try 10 seconds for now and if it fails again, then we will change it 15 seconds and then to 20 seconds.